docs/usage/key-concepts/dashboard.md

```diff
@@ -49,6 +49,19 @@ To disable the Dependency Dashboard, add the preset `:disableDependencyDashboard
 
 This section explains some common use cases where having the Dependency Dashboard can help.
 
+### Warnings for deprecated dependencies
+
+If deprecated packages are found - or any packages have community-sourced replacement PRs available - then Renovate will add a prominent warning about these packages near the top of the Dependency Dashboard.
+Here is an example of how this can look:
+
+> [!WARNING]
+> The following dependencies are deprecated:
+
+| Datasource | Name                | Replacement?                                                                      |
+| ---------- | ------------------- | --------------------------------------------------------------------------------- |
+| npm        | `airbnb-prop-types` | ![Available](https://img.shields.io/badge/available-green?style=flat-square)      |
+| npm        | `left-pad`          | ![Unavailable](https://img.shields.io/badge/unavailable-orange?style=flat-square) |
```

Comment:
The GitHub alert syntax won't display as you mean to in the published docs. I recommend putting the warning and table in an Material for MkDocs admonition block, with a prettier ignore line before the block.

I never tried a table in an admonition block, so the above may not work.

Or you can use the `>` characters to make a "quote" of the warning and table, and use a ⚠️ emoji.

https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage

Reply:
Would a screenshot be better?

Reply:
A screenshot will work well. Will need a descriptive alt text though. 🙂

Reply:
Actually I decided instead to remove the warning part and keep the rest

Reply:
That's a good way to go too. The warning styling is not critical to have in the docs. When users see the warning on the Dashboard they'll understand it right away. 😄 

Can you resolve this conversation for me then? 😉